pkg/plugins/yaml/main.go

```diff
@@ -15,13 +18,61 @@ var (
 
 // Yaml defines a specification for a "yaml" resource
 // parsed from an updatecli manifest file
-type Yaml struct {
-	File  string
-	Key   string
+type YamlSpec struct {
+	File string
+	Key  string
+	// Deprecated: use File instead
 	Path  string
 	Value string
 }
 
+// Yaml defines a resource of type "yaml"
+type Yaml struct {
+	Spec             YamlSpec
```

Comment:
Any reason to make public Spec instead of spec?

Reply:
Because of the resource of type `helm` which uses the `Yaml` types, as you can see in this change (diff of the current PR): https://github.com/updatecli/updatecli/pull/360/files#diff-beb8c6d04ce8efe2a8e014a3137114c2d8e1f9dd850ed2d5df13e40e3f685446R36

I chose to not work on decoupling both (like with `text`  / `file` `yaml`) in the context of this PR since it's a bug fix (and not a feature).

If it's ok for you, after fixing this bug,  #358 and #359 , I would like to factorize the logging logic of the yaml module (as in #354 for shell) , that would be a perfect spot for decoupling.